composer.json

```diff
@@ -43,5 +43,8 @@
         "psr-4": {
             "Solarium\\Tests\\": "tests/"
         }
+    },
+    "conflict": {
```

Comment:
ah actually this is not good I think. It will prevent those versions to be installed for anyone **using** solarium. We only need this for "dev" here...